src/utils/v8utils.h

```diff
@@ -20,18 +20,22 @@
 #include <nan.h>
 #include <string>
 #include <vector>
+#include <codecvt>
 
 namespace etcher {
 namespace v8utils {
-std::vector<std::string> GetArguments(v8::Local<v8::Array> arguments);
+std::vector<std::wstring> GetArguments(v8::Local<v8::Array> arguments);
 }  // namespace v8utils
 }  // namespace etcher
 
 #define YIELD_ERROR(CALLBACK, ERROR)                                           \
   {                                                                            \
-    v8::Local<v8::Value> argv[1] = {Nan::Error((ERROR))};                      \
-    Nan::MakeCallback(Nan::GetCurrentContext()->Global(), (CALLBACK), 1,       \
-                      argv);                                                   \
+    const uint16_t *message = (const uint16_t *)(ERROR).c_str();               \
```

Comment:
I wonder if it'd be better to only cast it inside the call to `NewFromTwoByte`, thus keeping it as `wchar_t` for as long as possible?
i.e. `Nan::Error(v8::String::NewFromTwoByte(isolate, (const uint16_t *)message))`